src/modules/project/projects.slice.js

```diff
@@ -14,6 +14,7 @@ const initialProjectState = {
     source: '',
   },
   selectedTag: {},
+  isMultipleFeaturesTaggingEnabled: false,
```

Comment:
So, there was initially a way to tag individual features from the detail page, but it did not make sense, because there is no UI for it in the detail page. That is when we had this term multiplefeaturetagging to represent tagging of more than one features at a time. 

For the redux related question, we toggle off and on the tags modal using this state variable. also, helps in avoiding a parameter in every method from component to usetags.